### PR TITLE
OEL-2794: Update components to run on Drupal 10.2.x

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -91,7 +104,7 @@ pipeline:
     when:
       event: tag
       matrix:
-        CORE_VERSION: "10.0.0"
+        CORE_VERSION: "10.1.0"
```
Comment (AaronGilMartinez) [inline .drone.yml:108]: Changed release condition, is this OK? Should we use 10.2 instead? I think they are the same

Comment (drishu) [reply .drone.yml:108]: its ok like this

File: README.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ Ideally this project should be managed with [Docker](https://www.docker.com/get-
 
 Check the [composer.json](composer.json) for required PHP version and other dependencies.
 
+#### Drupal 10.1
```
Comment (drishu) [inline README.md]: remove this as oe_content is a dev dependency

File: composer.json [hunk 3]
```diff
@@ -103,7 +103,19 @@
         },
         "_readme": [
             "Explicit requirement of nikic/php-parser ^4 as later versions are not compatible with grumphp @see https://github.com/phpro/grumphp/issues/1119"
-        ]
+        ],
+        "patches": {
```
Comment (drishu) [inline composer.json]: do we really need this in our project?

**Review by drishu — CHANGES_REQUESTED** [review]
we can move on to php 8.2 in docker compose for local development

File: README.md (hunk 2 above)
Comment (AaronGilMartinez) [reply README.md]: Removed

File: composer.json (hunk 3 above)
Comment (AaronGilMartinez) [reply composer.json]: Removed

**Review by drishu — APPROVED** [review]

**Pull request merged**